include/zephyr/mgmt/ec_host_cmd/ec_host_cmd.h

```diff
@@ -346,6 +347,20 @@ bool ec_host_cmd_send_in_progress_ended(void);
  * @retval The final status or EC_HOST_CMD_UNAVAILABLE if not available.
  */
 enum ec_host_cmd_status ec_host_cmd_send_in_progress_status(void);
+
+/**
+ * @brief Continue processing a handler in callback after returning EC_HOST_CMD_IN_PROGRESS.
+ *
+ * A Host Command handler may return the EC_HOST_CMD_IN_PROGRESS, but needs to continue work.
+ * This function should be called before returning EC_HOST_CMD_IN_PROGRESS with a callback that
+ * will be executed. The return status of the callback will be stored and can be get with the
+ * ec_host_cmd_send_in_progress_status function. The ec_host_cmd_send_in_progress_ended function
+ * can be used to check if the callback has ended.
+ *
+ * @retval EC_HOST_CMD_BUSY if any command is already in progress, EC_HOST_CMD_SUCCESS otherwise
```

Comment:
You should document the input parameters.  I thought there was a CI check for this?

Reply:
I think it was not checked because of `#ifdef CONFIG_EC_HOST_CMD_IN_PROGRESS_STATUS`, but not sure 100%.